src/libFLAC++/CMakeLists.txt

```diff
@@ -0,0 +1,49 @@
+set(prefix "${CMAKE_INSTALL_PREFIX}")
+set(exec_prefix "${CMAKE_INSTALL_PREFIX}")
+set(libdir "${CMAKE_INSTALL_FULL_LIBDIR}")
+set(includedir "${CMAKE_INSTALL_FULL_INCLUDEDIR}")
+configure_file(flac++.pc.in flac++.pc @ONLY)
+
+check_cxx_source_compiles("
+    #ifdef __STDC_NO_VLA__
+    syntax error;
+    #else
+    int fvla (int m, int * c)
+    {
+      int D[m];
+      return D[0] == c[0];
+    }
+
+    int main(int, char * []) { return 0; }
+    #endif"
+    HAVE_CXX_VARARRAYS)
+
+set(SOURCES
+    metadata.cpp
+    stream_decoder.cpp
+    stream_encoder.cpp)
+
+add_library(FLAC++-static STATIC ${SOURCES})
+target_compile_definitions(FLAC++-static PUBLIC FLAC__NO_DLL)
+target_include_directories(FLAC++-static INTERFACE
+    "$<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/include>"
+    "$<INSTALL_INTERFACE:include>")
```

Comment:
Common way is to use `GNUInstallDirs` module and `CMAKE_INSTALL_INCLUDEDIR`. User can override these options.

Reply:
Right. Thanks